riyaz/templates/course.html

```diff
@@ -83,7 +83,7 @@ <h2 id="instructors" class="mb-3">Instructors</h2>
     {% for instructor in course.get_instructors() %}
     <div id="author-{{ instructor.key }}" class="row mb-3">
         <div class="col-12 col-sm-4 col-md-3 col-lg-2 text-center">
-            <img class="rounded-circle" style="width: 100%; max-height: 128px; max-width: 128px;" src="{{ instructor.photo }}" />
+            <img class="rounded-circle" style="width: 100%; max-height: 128px; max-width: 128px;" src="{{ instructor.photo_path }}" />
```

Comment:
Shouldn't you use the asset url instead of just the photo_path? 

Now that we have assets, I think the `photo_path` can be replaced with `photo_id integer references asset`.

Reply:
But, `photo_path` need not be an asset being served by us.
It can be an external URL too.

Reply:
I suggest not get into storing external URLs. What happens if they external URL breaks or disappears? Who is responsible? 

Even if we want to support external URL, it should be abstracted out as assets and Course and Instructor should jjust see it as an asset either a local asset of remote. But I would suggest not to get into that complexity right away.

Reply:
Okay, then I think let's only support file paths now.
URLs can come later.

Reply:
Done.